python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -86,15 +88,18 @@ async def assert_feasibility(self):
                 setattr(self.mtcs.check, comp, False)
 
         # Check all enabled, liveliness and the m1m3 detailed state
+        self.expected_states = {DetailedState.PARKED, DetailedState.PARKEDENGINEERING}
```

Comment:
Is there a reason for this to be a class attribute? I don't think it is used anywhere else so you can drop the `self` and make it a local variable.

Reply:
I forgot to remove it, as I planned to use it in the cleanup method before you modified the `mtcs.stop_m1m3_bump_test` method.